bin/starred_search

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env node
-/******/ (() => { // webpackBootstrap
+#!/usr/bin/env node
/******/ (() => { // webpackBootstrap
```

Comment:
Can you please revert this file and push again? I don't want to add this to git ignore because the release workflow builds this and pushes it to the repo, so we need to be able to stage it.